ansible/workers.yml

```diff
@@ -125,11 +125,21 @@
   - name: Ensure SCAP Security Guide dependencies are installed (Fedora only)
     package:
       name:
-      - python2-jinja2  # needed for the new SSG yaml/jinja2 port
-      - python2-pytest  # needed for unit tests + coverage
+      - python3-pytest  # needed for unit tests + coverage
+      - python3-jinja2
+      - python3-PyYAML
       state: installed
     when: ansible_distribution == 'Fedora'
 
+  - name: Ensure Fedora builds are as Python3-only as possible.
+    package:
+      name:
+      - python2-jinja2
+      - python2-pytest
+      - python2-pyyaml
+      state: absent
+    when: ansible_distribution == 'Fedora'
```

Comment:
this will break everything

Reply:
You are right, they are required by Ansible, and some other packages.